apps/src/applab/dropletConfig.js

```diff
@@ -31,7 +31,7 @@ var stringBlockPrefix = 'str.';
 // Configure shared APIs for App Lab
 // We wrap this because it runs before window.Applab exists
 function applabExecuteCmd(...args) {
-  Applab.executeCmd.call(Applab, ...args);
+  return Applab.executeCmd.call(Applab, ...args);
```

Comment:
Was alarmed to discover that we were dropping App Lab command return values here.  This didn't show up because this helper is only used when we need to inject the `executeCmd` method into shared APIs, like Maker - and none of those shared APIs were using return values yet! (The `digitalRead` and `analogRead` maker methods are actually native-async methods masquerading as synchronous in App Lab, so they use callbacks).